lib/person.rb

```diff
@@ -11,6 +11,12 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rental << rental
+    rental.person = self
   end
```

Comment:
-  You may kindly refer to the comment above and fix this method.

Reply:
Thanks fixed all.